jvm-libs/extensions/build.gradle

```diff
@@ -16,3 +16,8 @@
 plugins {
   id 'maru.kotlin-library-conventions'
 }
+
+dependencies {
+  api "build.linea.internal:futures"
+  api "tech.pegasys.teku.internal:executionclient"
```

Comment:
TBH, exposing these as `api` doesn't seem like a good idea, as it will blur the dependencies. I would rather avoid transitive dependencies on `build.linea.internal:futures` and import them explicitly as `implementation` where they're needed